libgraph/src/analytics/bfs/bfs.cpp

```diff
@@ -159,7 +159,7 @@ AsynchronousAlgo(
   katana::GAccumulator<size_t> bad_work;
   katana::GAccumulator<size_t> WL_empty_work;
 
-  (*node_data)[source] = 0;
+  (*node_data)[source.value()] = 0;
```

Comment:
This is problematic, right? `source.value` are not contiguous and `size of node_data == num_nodes.`

Reply:
Yes, it will be a problem when we switch to LSPG. Not a problem today, obviously.